forge-game/src/main/java/forge/game/GameEntityCounterTable.java

```diff
@@ -157,8 +157,13 @@ public void replaceCounterEffect(final Game game, final SpellAbility cause, fina
 
             // Apply counter after replacement effect
             for (Map.Entry<Optional<Player>, Map<CounterType, Integer>> e : values.entrySet()) {
+                boolean remember = cause.hasParam("RememberPut");
```

Comment:
`boolean remember = cause != null && cause.hasParam("RememberPut");`

Reply:
Thanks; log was not helpful this time around

This makes sense

I'll be a programmer yet